keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,418 @@
+- Some critical windows functionality, such as the ability to run activeDirectory as an identity provider for a pod, need to be well-defined in a Kubernetes context for individuals planning on integrating production Windows workloads into their Kubernetes support model.
+- The introduction of "Privileged containers" (formally referred to as `HostProcess` pods), further differentiates Linux from Windows in subtle ways which specifically will, eventually, allow Windows users to implement many of the common idioms in Kubernetes using a comparable, but different runtime paradigm.
+- The Kubernetes ServiceProxy implementation is not at parity with that of Linux, and likely never will be.  This needs to be made explicit so that attempts to reach parity between Kuberentes service API implementations on Windows and Linux can be approached in a systematic manner in terms of core requirements and supportability.  Features such as DSR (direct-server-return), EndpointSlices, and so on generally require special treatment for Windows clusters, some of which may have a complex interplay with the underlying CNI provider.
+
+We propose the following definitions as the Windows conformance standard for Kubernetes clusters that are released after Kuebrentes 1.23 and onwards.  This specification is to be updated as new features are added, or removed, from the Kubernetes API itself.
+
+### User Stories (Optional)
+
+#### Story 1
+
+As a Windows Application developer I want to verify that the features i rely on in Kubernetes are supported on my specific K8s Windows cluster.
+
+#### Story 2
+
+As a Kubenetes vendor I want to evalaute the completeness of my Kubernetes support matrix in context of Windows supported features in the Kubernetes API.
+
+#### Story 3
+
+As a Kubernetes developer I want to immediately know wether or not the features I want to add will effect the Windows support matrix, and if so, I want to be able to rapidly convey this in a canonical way to the broader community.
+
+#### Story 4
+
+As an IT administrator, I want to know wether my current version of Kubernetes supports ActiveDirectory
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate? Think broadly.
+For example, consider both security and how this will impact the larger
+Kubernetes ecosystem.
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+
+Consider including folks who also work outside the SIG or subproject.
+-->
+
+## Design Details
+
+### Specification of Windows Core Conformance (Kubernetes 1.23)
+
+- In order to qualify as a Windows test, the pods that demonstrate functionality for a given attribute of a cluster much be running on a `pause` image, and this image must be scheduled to an operating system that:
+
+    - 1809
+    - 1909
```

Comment:
Why are we considering 1909 and 2004 for Windows Conformance? 1909 SAC has already reached it's end of service and 2004 SAC is rapidly approaching it's end of service at the end of this year. From what I understand, Microsoft is dropping SAC versions entirely in the coming future and will be LTSC only.

I believe we should only be performing conformance tests on what would be officially supported at the time of 1.23, which would be LTSC versions of Windows Server (Windows Server 2019 and Windows Server 2022).

Furthermore, we should consider switching to identifying Windows Server 2019 as something other than 1809 as this will continue to be a source of confusion as this build version refers to both an LTSC and SAC release.

Reply:
per conversation in sig-windows @perithompson / ross to push updates for this 

2 latest SACs
latest LTS

Reply:
2004 -> out per muzz

Reply:
curate a table as part of this KEP , updated from version to version

Reply:
Do we need to have some way to define Windows OS patches that are required for Operational Conformance?

Reply:
removed since 2022 simplifies things